core/src/main/java/com/datastax/oss/driver/api/core/metadata/NodeProperties.java

```diff
@@ -0,0 +1,29 @@
+/*
+ * Copyright DataStax, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.datastax.oss.driver.api.core.metadata;
+
+import com.datastax.oss.driver.api.core.Version;
+
+/** The keys for the additional properties stored in {@link Node#getExtras()}. */
+public class NodeProperties {
```

Comment:
Given that this is kind of temporary (we'll use `DseNodeProperties` after we merge the drivers), I'd rather put this in an internal package.

Reply:
Makes sense, I'll move it to an equivalent spot in the internal space